src/elements/stepper/stepper/stepper.ts

```diff
@@ -29,6 +30,22 @@ export
 class SbbStepperElement extends SbbHydrationMixin(LitElement) {
   public static override styles: CSSResultGroup = style;
 
+  /**
```

Comment:
Could you create a test case covering the fix?

Reply:
yes, sure, i'll update the pr soonish